app/views/layouts/_social_meta.html.haml

```diff
@@ -1,5 +1,5 @@
-%meta{name: 'title', content: 'hackEDU'}
+%meta{name: 'title', content: 'Hack Club'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace